server/agent_config/example.yaml

```diff
@@ -1440,49 +1440,52 @@ vtap_group_id: g-xxxxxx
 
     ## Ebpf reorder cache size
     ## Default: 16. Range: [8, 128]
+    ## Note: Configure this parameter based on the unordered span size.
     #syscall-out-of-order-cache-size: 16
```

Comment:
```
## eBPF Reorder Cache Size
## Default: 16. Range: [8, 128]
## Note: When `syscall-out-of-order-reassembly` is enabled, up to `syscall-out-of-order-cache-size` eBPF socket events (each event consuming up to `l7_log_packet_size` bytes) will be cached in each TCP/UDP flow to prevent out-of-order events from impacting application protocol parsing. Since eBPF socket events are sent to user space in batches, out-of-order scenarios mainly occur when requests and responses within a single session are processed by different CPUs, causing the response to reach user space before the request.
```

换行你处理下。另外下面的也要改，等我改完